synapse/lib/aha.py

```diff
@@ -184,6 +185,8 @@ async def addAhaSvc(self, name, info, network=None):
         NOTE: In order for the service to remain marked "up" a caller
               must maintain the telepath link.
         '''
+        if network is not None:
+            s_common.deprecated('addAhaSvc() network argument', curv='v2.206.0')
```

Comment:
did we want to specify a depr version for these as well?

Reply:
We default to using 3.0.0.